bobweb/bob/test_command_gpt.py

```diff
@@ -136,32 +137,18 @@ async def test_should_contain_correct_response(self):
         expected_reply = 'gpt answer'
         self.assertEqual(expected_reply, chat.last_bot_txt())
 
-    async def test_should_use_default_model_when_assigned_so_by_random(self):
+    async def test_should_use_default_model_when_assigned_so(self):
         chat, user = init_chat_user()
         mock_method = AsyncMock(return_value=MockLiteLLMResponseObject())
         with (
-            mock.patch('bobweb.bob.litellm_utils.litellm.acompletion', mock_method),
-            mock.patch('random.random', return_value=0.51)
+            mock.patch('bobweb.bob.litellm_utils.litellm.acompletion', mock_method)
         ):
             await user.send_message('/gpt foo')
             mock_method.assert_called_with(
-                model='openai/gpt-4o',
+                model='openai/gpt-5',
```

Comment:
Now that I think of it, it would make sense to have the model name in one constant in test classes and have each test case reference that so there would not be need to update the model number to each test case each time the default model is changed.